packages/ses/src/lockdown-shim.js

```diff
@@ -53,7 +53,8 @@ import { makeCompartmentConstructor } from './compartment-shim.js';
 
 const { details: d, quote: q } = assert;
 
-let firstOptions;
+/** @type {Error?} */
+let priorLockdown = null;
```

Comment:
Why `null` rather than `undefined`?
```suggestion
let priorLockdown;
```
I don't have any strong reason for preferring one over the other. But when there is no other reason, I generally default to `undefined`.